tools/test/audio/process_test.m

```diff
@@ -409,6 +409,9 @@
 	test.thdnf_max = [];            % Set per component
 	test.dr_db_min = 80;            % Min. DR
 	test.fr_rp_max_db = 0.5;        % Allow 0.5 dB frequency response ripple
+
+	% No need to collect trace
+	test.trace = '';
```

Comment:
At line number 101, `tn = 1;`, tn is not used; please consider using it or removing it.

At line number 321, `min_bits = min(test.bits_in, test.bits_out);`, do you intend to use minimum bits elsewhere ? Otherwise, it's unnecessary code that should be removed.

Reply:
@ShriramShastry how is this related to this PR?

Reply:
It has nothing to do with PR and does not rely on PR to succeed. A little correction can help greatly improve code quality, and I believe Seppo won't mind (because he wrote the majority of the code) considering the proposal.

Reply:
These look like interesting suggestions but they're still off-topic here.

Reply:
Hmm - sure it's off the  topic, what are your thoughts if you notice improvement and believe that rectification would be beneficial without adding extra work?  I am not sure if the chosen approach is OK or NOT, and for the PR, I request Seppo to consider but leave the decision to him.

Reply:
Thanks for suggestions. They are better to leave to another PR later. If you like you are welcome to make a PR since you found them. Or I can do later when time, as you prefer.

Reply:
I would want to propose that you incorporate it as well as as you can take over [https://github.com/thesofproject/sof/pull/6056](url).
Thank you!!

Reply:
https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html